mungegithub/mungers/close-stale.go

```diff
@@ -109,14 +98,21 @@ func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool)
 	return lastHuman, true
 }
 
-func findLastHumanIssueUpdate(obj *github.MungeObject) (*time.Time, bool) {
-	lastHuman := obj.Issue.CreatedAt
+func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	pr, ok := obj.GetPR()
+	if !ok {
+		return nil, ok
+	}
 
-	comments, ok := obj.ListComments()
+	comments, ok := obj.ListReviewComments()
 	if !ok {
 		return nil, ok
 	}
 
+	return findLastHumanPullRequestUpdateHelper(comments, pr.CreatedAt)
```

Comment:
I'm not super excited about the `Helper` prefix. Could you change the `close-stale` munger to get `Comments`/`ReviewComments` in its own `Munge` function so that we don't need the non-helper and the helper function?
Also, I'm not super fan of coupling both mungers like that.